httpbin/cmd/cmd_test.go

```diff
@@ -431,6 +464,28 @@ func TestLoadConfig(t *testing.T) {
 				MaxBodySize:            httpbin.DefaultMaxBodySize,
 				MaxDuration:            httpbin.DefaultMaxDuration,
 				AllowedRedirectDomains: []string{"foo", "bar", "baz"},
+				LogFormat:              testDefaultLogFormat,
+			},
+		},
+		"ok use json log format": {
+			args: []string{"-log-format", "json"},
+			env:  map[string]string{"LOG_FORMAT": "json"},
+			wantCfg: &config{
+				ListenHost:  "0.0.0.0",
+				ListenPort:  8080,
+				MaxBodySize: httpbin.DefaultMaxBodySize,
+				MaxDuration: httpbin.DefaultMaxDuration,
+				LogFormat:   "json",
+			},
+		},
+		"ok use text log format": {
+			args: []string{"-log-format", "text"},
+			wantCfg: &config{
+				ListenHost:  "0.0.0.0",
+				ListenPort:  8080,
+				MaxBodySize: httpbin.DefaultMaxBodySize,
+				MaxDuration: httpbin.DefaultMaxDuration,
+				LogFormat:   testDefaultLogFormat,
```

Comment:
No, just this specific one, where we're very explicitly testing a specific value.

Reply:
Got it thanks for clarification 🙏

Reply:
… but thinking about it just a bit more, it would probably make sense to define this as a real default value in `cmd.go` the same way we do with these:
https://github.com/mccutchen/go-httpbin/blob/874932b8a03a74758ac8ef61439fcd4a0550211f/httpbin/cmd/cmd.go#L25-L26

Then, that real default var can be referenced here in the test suite rather than this separate `testDefaultLogFormat` var.

(This specific test should still test the specific value without using the variable, though.)

Reply:
Yeah it definitely makes sense, I did the change on my local 👍